src/shogun2-core/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,66 @@
+#################################################
+# LOGGING PROPERTIES FOR TESTING PURPOSE
```

Comment:
As we have migrated to log4j v2 with one of the latest merges, we should replace this config with an appropriate log4j v2 config xml. Just have a look here:

* https://github.com/terrestris/shogun2/blob/master/src/shogun2-core/src/test/resources/log4j2.xml
* https://github.com/terrestris/shogun2/blob/master/src/shogun2-webapp-archetype/src/main/resources/archetype-resources/src/main/resources/log4j2.xml

Reply:
Since @hwbllmnn already introduced a corresponding log4j2 config in the test resources, I've simply removed `log4j.properties` in 42ff914